views/score.erb

```diff
@@ -0,0 +1,18 @@
+<h1>This is the score page.</h1>
+
+
+<form action="" method="post" accept-charset="utf-8">
+  <fieldset>
+    <legend>Word to score</legend>
+    <label><input type="text" name="word" value="" id="word"></label>
+  </fieldset>
+</form>
+
+<% unless params == {} %>
+  <% if @final_score.nil? %>
+    <%= erb :error_partial %>
+  <% else %>
+    <h2>Word: <%= params[:word] %>, Score: <%= @final_score %></h2>
+    <%= erb :letter_value_partial, :locals => { :k => params[:word]} %>
```

Comment:
I'd recommend a more explicit variable name rather than `k` because when you get into the `letter_value_partial` then there is no context about what kind of data this variable contains.